### PR TITLE
Gui 5 widget cleanup

### PR DESCRIPTION
**Pull request by retiutut**
Clean up Focus, SSVEP,  and Presentation mode

Piggy-backs on #759 from @daniellasry

### CODE REVIEW

File: OpenBCI_GUI/ControlPanel.pde [hunk 1]
```diff
@@ -1985,7 +1985,7 @@ class SampleRateCytonBox {
         fill(bgColor); //set color to green
         textFont(h3, 16);
         textAlign(LEFT, TOP);
-        text("  " + str(getSampleRateSafe()) + "Hz", x + padding + 142, y + padding); // print the channel count in green next to the box title
+        text("  " + str(currentBoard.getSampleRate()) + "Hz", x + padding + 142, y + padding); // print the channel count in green next to the box title
```
Comment (Andrey1994) [inline OpenBCI_GUI/ControlPanel.pde:1988]: its a little unclear to me how it works... I think in ControlPanel there is no currentBoard object(its set to BoardNull) and only when we close controlPanel we create correct instance

Comment (daniellasry) [reply OpenBCI_GUI/ControlPanel.pde:1988]: It doesn't work you are correct. That text field should simply be removed.
image

Comment (daniellasry) [reply OpenBCI_GUI/ControlPanel.pde:1988]: removed in upstream change

Comment (daniellasry) [reply OpenBCI_GUI/ControlPanel.pde:1988]: Deleted in upstream change

**Pull request merged**